### PR TITLE
Adding more quotes.

### PR DESCRIPTION
**Pull request by enjiniya-sama**
I really like your project,  just trying to contribute something.

### CODE REVIEW

File: resources/quotes.json [hunk 1]
```diff
@@ -1,4 +1,34 @@
...
+  {
+      "text": "Every time someone puts an objection to me, I want to say: 'OK, OK, let's go on to something else.' Objections have never contributed anything.",
+      "source": "Dialouges (1987)",
+      "numChars": 143
+  },
+  {
+      "text": "It is of the nature of Reason to perceive things under a certain species of eternity.",
+      "source": "Ethics, Spinoza",
+      "numChars": 111
+  },
```
Comment (rmehri01) [inline resources/quotes.json:16]: I got 85 chars from this, perhaps part is missing or was just entered wrong?

**Review by rmehri01 — COMMENTED** [review]
Thanks, I'm glad you like it! Any little bit helps :)

File: resources/quotes.json (hunk 1 above)
Comment (enjiniya-sama) [reply resources/quotes.json:16]: Oh yes , I removed some latin text but forgot to update numChars . Terribly sorry.

Comment (rmehri01) [reply resources/quotes.json:16]: No problem! Can you just change it quickly when you have time?

Comment (enjiniya-sama) [reply resources/quotes.json]: Done.

**Review by rmehri01 — APPROVED** [review]
Looks good, thanks again!

**Pull request merged**